ls1intum/Artemis
# [Exam] Track started exam and submission date

### Checklist

### Motivation and Context
We want to track if the student has started and submitted the exam and when he handed in.

### Description
- [x] I've added a `started` flag to StudentExam that will be bet set when to `true` when the student clicks the *Start* button (loads the student exam)
- [x] I've added a `submissionDate` attribute that will be set when the student submits the exam
- [x] I've added the two new attributes and the `submitted` flag to the student exam overview
- [x] I've added the two new attributes and the `submitted` flag to the student exam detail view

### Steps for Testing
1. Log in to Artemis
2. Participate in an exam (the easiest way to test it is with your (instructor/admin) account)...
3. Click the *Start* button of the exam
4. Check in the exam management that *yes* is shown next to *Started* in the student exam overview and detail view
5. Hand in the exam early and click the *Finish* button
6. Check in the exam management that *yes* is shown next to *Submitted* in the student exam overview and detail view
7. Check in the exam management that the submission date is shown in the student exam overview and detail view

### Screenshots
![image](https://user-images.githubusercontent.com/15168372/87575838-cf33d800-c6d0-11ea-9c5b-26de31b9dd98.png)
![image](https://user-images.githubusercontent.com/15168372/87575883-de1a8a80-c6d0-11ea-9c2e-cff4b1b651e6.png)
